src/test/java/jp/co/yahoo/yosegi/message/formatter/json/TestJacksonMessageWriter.java

```diff
@@ -0,0 +1,288 @@
+  }
+
+  @Test
+  public void T_createObj_2() throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    JacksonMessageWriter writer = new JacksonMessageWriter();
+    PrimitiveObject obj = null;
+    byte[] result = writer.create( obj );
+    JsonNode node = mapper.readTree( result );
+    assertTrue( ( node instanceof NullNode ) );
+  }
+
+  @Test
+  public void T_createArray_1() throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    JacksonMessageWriter writer = new JacksonMessageWriter();
+    List<Object> list = Arrays.asList( "a" , "b" , "c" );
+    byte[] result = writer.create( list );
+    JsonNode node = mapper.readTree( result );
+    assertTrue( ( node instanceof ArrayNode ) );
+
+    ArrayNode arrayNode = (ArrayNode)node;
+    assertEquals( arrayNode.size() , 3 );
+    assertTrue( arrayNode.get(0).isTextual() );
+    assertTrue( arrayNode.get(1).isTextual() );
+    assertTrue( arrayNode.get(2).isTextual() );
+    assertEquals( arrayNode.get(0).asText() , "a" );
+    assertEquals( arrayNode.get(1).asText() , "b" );
+    assertEquals( arrayNode.get(2).asText() , "c" );
+  }
+
+  @Test
+  public void T_createArray_2() throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    JacksonMessageWriter writer = new JacksonMessageWriter();
+    List<Object> list = null;
+    byte[] result = writer.create( list );
+    JsonNode node = mapper.readTree( result );
+    assertTrue( ( node instanceof ArrayNode ) );
+
+    ArrayNode arrayNode = (ArrayNode)node;
+    assertEquals( arrayNode.size() , 0 );
+  }
+
+  @Test
+  public void T_createMap_1() throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    JacksonMessageWriter writer = new JacksonMessageWriter();
+    Map<Object,Object> map = new HashMap<Object,Object>();
+    map.put( "key1" , "a" );
+    map.put( "key2" , "b" );
+    map.put( "key3" , "c" );
+    byte[] result = writer.create( map );
+    JsonNode node = mapper.readTree( result );
+    assertTrue( ( node instanceof ObjectNode ) );
+
+    ObjectNode objNode = (ObjectNode)node;
+    assertEquals( objNode.size() , 3 );
+    assertTrue( objNode.get("key1").isTextual() );
+    assertTrue( objNode.get("key2").isTextual() );
+    assertTrue( objNode.get("key3").isTextual() );
+    assertEquals( objNode.get("key1").asText() , "a" );
+    assertEquals( objNode.get("key2").asText() , "b" );
+    assertEquals( objNode.get("key3").asText() , "c" );
+  }
+
+  @Test
+  public void T_createMap_2() throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    JacksonMessageWriter writer = new JacksonMessageWriter();
+    Map<Object,Object> map = null;
+    byte[] result = writer.create( map );
+    JsonNode node = mapper.readTree( result );
+    assertTrue( ( node instanceof ObjectNode ) );
+
+    ObjectNode objNode = (ObjectNode)node;
+    assertEquals( objNode.size() , 0 );
+  }
+
+  @Test
+  public void T_createParser_1() throws IOException {
```

Comment:
Please write the test method name is to describe what is wanted to in the test instead of using number (it makes vague one).

Reply:
I changed the method name.